README.md

```diff
@@ -25,8 +25,13 @@ To know more about Zeppelin, visit our web site [http://zeppelin.incubator.apach
 ## Getting Started
 
 ### Before Build
-If you don't have requirements prepared, install it. 
-(The installation method may vary according to your environment, example is for Ubuntu.)
+If you don't have requirements prepared, you will need to install them.  
+
+A script to launch a virtual machine with all of the requirements is also available 
```

Comment:
seems like this should be a different subsection?
also you could make /scripts/vagrant/zeppelin-dev/README.md into doc and linked from http://zeppelin.incubator.apache.org/docs/0.5.5-incubating/docs.html